src/corelibs/U2View/src/ov_msa/ScrollController.h

```diff
@@ -98,6 +98,11 @@ class U2VIEW_EXPORT ScrollController : public QObject {
 
     /** Called right after zoom-on/out/reset or any other font change operation to update internal scrollbars scales. */
     void updateScrollBarsOnFontOrZoomChange();
+    void setHScrollBarVisible(bool visible);
```

Comment:
Can we make these methods private? It is responsibility of the ScrollController (internal logic) to make scroll-bars visible or not right?

Reply:
We can't
Multiline scroll controller need to explicitly hide all lines' scrollbars